.npmignore

```diff
@@ -1,6 +1,7 @@
 .nyc_output/
 examples/
 flow-typed/
+media/
```

Comment:
Rather than in media, please place in root of the project next to dotenv.png.

Reply:
np.